docs/tutorials/cloud-shell.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Cloud Shell
+---
+
+The cloud shell feature helps you to manage the application via CLI(vela、kubectl). such as:
+
+* Creating applications in batches with the YAML.
+* Watch the logs of the application.
+* Open the shell TTY of the application instance.
+* Upload and download files from the shell environment.
+* Others...
+
+### Enable the addon
+
+This operation needs to have the permission of the addon management.
+
+```bash
+vela addon enable cloudshell
+```
+
+### Open the cloud shell environment
+
+![open-cloud-shell](https://static.kubevela.net/images/1.5/cloud-shell.jpg)
+
+On the UI top right, there is a cloud shell button, user clicks it will open the cloud shell environment. By default, all users have permission to open the cloud shell. If you want to disable this feature for some users, you could create a new platform role that has `Disable CloudShell` permission. Bind this role to the user, who you want to disable.
+
+![cloud shell environment](https://static.kubevela.net/images/1.5/cloud-shell-environment.jpg)
+
+Connect to the cloud shell environment, you could exec `vela --help` to get the help message. In the console, users have some permissions(Kubernetes RBAC) that auto grant base the permission in the UI. there are some rules:
```

Comment:
After connected to the cloud shell environment,

auto-grant the base permission